tests/user.test.js

```diff
@@ -0,0 +1,30 @@
+import chai from 'chai';
```

Comment:
@ssanusi Integration tests and unit tests are yet to be done

Reply:
@chikeozulumba its work in progress thanks for pointing that out i have to figure out how the unit test will be implemented